src/cuga/backend/server/manage_routes/knowledge_routes.py

```
@@ -0,0 +1,464 @@
+        state = draft_state(request)
+        if state:
+            try:
+                base_agent_id_for_key = _parse_agent_id(str(agent_id))
+                cfgs = getattr(state, "draft_knowledge_configs", None)
+                if not isinstance(cfgs, dict):
+                    cfgs = {}
+                    state.draft_knowledge_configs = cfgs
+                cfgs[base_agent_id_for_key] = validated
+                state.draft_knowledge_config = validated
+            except Exception:
+                pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)